### ISSUE TITLE
[4] Update nightly build "Call to a member function attributes() on null"

### ISSUE THREAD

**Issue by ReLater** [joomla/joomla-cms#28421]
I don't know if it's of interest.

### Steps to reproduce the issue
- Update a nightly build of last week via "Joomla Update" > "Live Update" > "Reinstall" with "Update Source" update.joomla.org/core/nightlies/next_major_list.xml

### Actual result
Call to a member function attributes() on null 
```
Call stack
--
# | Function | Location
1 | () | JROOT/libraries/src/Installer/Adapter/PluginAdapter.php:424
2 | Joomla\CMS\Installer\Adapter\PluginAdapter->setupUninstall() | JROOT/libraries/src/Installer/InstallerAdapter.php:1180
3 | Joomla\CMS\Installer\InstallerAdapter->uninstall() | JROOT/libraries/src/Installer/Installer.php:764
4 | Joomla\CMS\Installer\Installer->uninstall() | JROOT/administrator/components/com_admin/script.php:412
5 | JoomlaInstallerScript->uninstallRepeatableFieldsPlugin() | JROOT/administrator/components/com_admin/script.php:96
6 | JoomlaInstallerScript->update() | JROOT/administrator/components/com_joomlaupdate/src/Model/UpdateModel.php:866
7 | Joomla\Component\Joomlaupdate\Administrator\Model\UpdateModel->finaliseUpgrade() | JROOT/administrator/components/com_joomlaupdate/src/Controller/UpdateController.php:180
8 | Joomla\Component\Joomlaupdate\Administrator\Controller\UpdateController->finalise() | JROOT/libraries/src/MVC/Controller/BaseController.php:728
9 | Joomla\CMS\MVC\Controller\BaseController->execute() | JROOT/libraries/src/Dispatcher/ComponentDispatcher.php:146
10 | Joomla\CMS\Dispatcher\ComponentDispatcher->dispatch() | JROOT/libraries/src/Component/ComponentHelper.php:384
11 | Joomla\CMS\Component\ComponentHelper::renderComponent() | JROOT/libraries/src/Application/AdministratorApplication.php:131
12 | Joomla\CMS\Application\AdministratorApplication->dispatch() | JROOT/libraries/src/Application/AdministratorApplication.php:174
13 | Joomla\CMS\Application\AdministratorApplication->doExecute() | JROOT/libraries/src/Application/CMSApplication.php:231
14 | Joomla\CMS\Application\CMSApplication->execute() | JROOT/administrator/includes/app.php:63
15 | require_once() | JROOT/administrator/index.php:36
```

**Comment by SharkyKZ** [comment]
Should have been fixed with https://github.com/joomla/joomla-cms/pull/28399.

**Comment by ReLater** [comment]
> Should have been fixed with #28399.

Hm, 28399 should be already in the nightly build of today. I tried also with copying the full package before.

I'll try with a new installation and try again the Live Update faeture.
